lib/model/query/analytics.js

```diff
@@ -600,6 +638,17 @@ const previewMetrics = () => (({ Analytics }) => Promise.all([
 
   metrics.system.uses_external_db = Analytics.databaseExternal(config.get('default.database.host'));
 
+  // 2023.4.0 metrics
+  metrics.system.num_client_audit_attachments = caAttachments;
+  metrics.system.num_client_audit_attachments_failures = caFailures;
+  metrics.system.num_client_audit_rows = caRows;
+  metrics.system.num_audits_failed = failedAudits.any_failure;
+  metrics.system.num_audits_failed5 = failedAudits.failed5;
+  metrics.system.num_audits_unprocessed = failedAudits.unprocessed;
+
+  // TODO for 2023.4
+  metrics.system.sso_enabled = 0;// (something)
```

Comment:
I think this could come in a follow-up PR once #910 is merged, if that makes things easier.

Reply:
#910 is merged, so it should be possible to calculate this metric now. 🎉